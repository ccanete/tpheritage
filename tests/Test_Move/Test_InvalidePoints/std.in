MOVE test
MOVE test 5
EXIT