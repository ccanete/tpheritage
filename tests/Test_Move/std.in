R test 0 0 2 2
MOVE test 5 5
EXIT