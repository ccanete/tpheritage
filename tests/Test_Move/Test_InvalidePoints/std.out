ERR
# Invalide values.
ERR
# Invalide values.

# The program is being closed.